src/spikeinterface/postprocessing/tests/common_extension_tests.py

```diff
@@ -76,6 +77,16 @@ def setUp(self):
             overwrite=True,
         )
         self.we2 = we2
+
+        # make we read-only
+        if platform.system() != "Windows":
```

Comment:
Does the test works in windows if this statements is not run? I am wondering if what should be skipped is the test itself if we rely chmod. As @zm711 has pointed out that this is not reliable on windows.

Reply:
Altought the core tests are running on windows so...

Reply:
My point is not that it won't work just that it may introduce a hard to detect bug in the future since it does not behave as the user intends. So if you ever have a test that address writability, Windows behavior would diverge from Linux/Mac. If all the tests are only testing the readability, then they wouldn't "experience" the difference in the file identities. So that's why I said future problem since it may in the future add a hard to detect bug, but won't definitely introduce one.